src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -885,5 +944,17 @@ private struct LibraryDependency
             public string Name { get; set; }
             public NuGetVersion MinVersion { get; set; }
         }
+
+        private class ModifiableRuntimeLibrary
+        {
+            public RuntimeLibrary Library { get; set; }
+            public HashSet<string> Dependents { get; set; }
```

Comment:
I think it's easy to get confused about what "dependent" means (libraries that depend on this one, ie the opposite of a dependency), so it would be good ta have a clarifying comment here.